src/app.ts

```diff
@@ -98,16 +99,14 @@ export async function initSlack(): Promise<void> {
 }
 
 export async function initDiscord(): Promise<void> {
+  const discordBotToken = await Config.findOne('discordBotToken');
```

Comment:
When fetching for a config item and you know the name, you should use `getValueAs` so we can have better type safety

```suggestion
  const discordBotToken = await Config.getValueAs('discordBotToken', 'string', false);
```